src/SFA.DAS.ProviderApprenticeshipsService.Web/Controllers/CommitmentController.cs

```diff
@@ -56,7 +56,7 @@ public async Task<ActionResult> AgreementNotSigned(long providerId, string hashe
             model.RequestListUrl = Url.Action(redirectTo, new { providerId });
 
             if (model.IsSignedAgreement)
-                return RedirectToAction("Details", new { providerId, hashedCommitmentId });
+                return Redirect(_providerUrlhelper.ProviderCommitmentsLink($"{providerId}/unapproved/{hashedCommitmentId}/details"));
```

Comment:
Do we have anything in the backlog to move this controller action over to das-providercommitments?